shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -89,12 +124,12 @@ public String getDescription() {
         + "Where category is an optional argument, if no arguments passed in, "
         + "summary information will be printed out."
         + "category can be one of the following:\n"
-        + "    summary          Summarized Alluxio cluster information\n";
+        + "    summary          Summarized Alluxio cluster information";
   }
 
   @Override
   public void validateArgs(String... args) throws InvalidArgumentException {
-    if (args.length > 1) {
+    if (args.length > 2) {
```

Comment:
isn't there only one arg, summary?

Reply:
I considered future commands, I will change it back to 1, thanks